### ISSUE TITLE
ocaml-width

### ISSUE THREAD

**Issue by objmagic** [seliopou/ocaml-fullwidth#2]
since we offer API that converts fullwidth back to halfwidth, shall we rename this library to "ocaml-width"?

**Comment by seliopou** [comment]
Agreed that the current name isn't great but "width" is even more ambiguous.

How about unicode-width?
